runtime/compiler/p/runtime/PicBuilder.spp

```diff
@@ -64,6 +64,22 @@
 #define maskVFT(reg)    rlwinm reg, reg, 0, 0, 31-J9TR_RequiredClassAlignmentInBits
 #endif
 
+#if defined(OMR_GC_COMPRESSED_POINTERS)
+#if defined(OMR_GC_FULL_POINTERS)
+#define LOAD_CLASS(dest, src) \
+	laddr dest,J9TR_VMThreadCompressObjectReferences(r15) ; \
+	cmpli cr0, CmpAddr, dest, 0 ; \
+	beq $+12 ; \
+	lwz dest, J9TR_J9Object_class(src) ; \
+	b $+8 ; \
+	laddr dest, J9TR_J9Object_class(src)
```

Comment:
rather having a taken branch always, we can have a sequence of no taken branch for one way:

laddr  dest, J9TR_VMThreadCompressObjectReferences(r15) 
cmpli cr0, CmpAddr, dest, 0 
laddr dest, J9TR_J9Object_class(src)
beq  +8
lwz dest, J9TR_J9Object_class(src)

Reply:
Fixed.

Reply:
Verified that mixed mode still works after this change.